pkgs/build-support/emacs/wrapper.nix

```
@@ -59,7 +59,7 @@ runCommand
   (appendToName "with-packages" emacs).name
   {
     nativeBuildInputs = [ emacs lndir makeWrapper ];
-    inherit emacs explicitRequires;
+    inherit emacs explicitRequires lndir;
```

```suggestion
    inherit emacs explicitRequires;
```
that goes to nativeBuildInputs only